linter/rules/field_requires_description.py

```diff
@@ -7,6 +7,6 @@ def applies_to() -> Tuple[str, ...]:
         return ('dimension', 'measure')
 
     def run(self, field: Any) -> bool:
-        if not 'description' in field:
+        if not 'description' in field and field.get('hidden') != 'yes':
```

Comment:
incredibly minor but makes a bit more sense to switch these conditions:

```
if field.get('hidden') != 'yes' and not 'description' in field
```

since we don't even want to check for the description field if the hidden property is yes.

Reply:
Yeah, that's a great point. Thanks for flagging